packages/block-editor/src/hooks/margin.js

```diff
@@ -167,33 +173,32 @@ export function MarginEdit( props ) {
 	} );
 }
 
-export function MarginVisualizer( { clientId, attributes } ) {
+export function MarginVisualizer( { clientId, attributes, isMouseOver } ) {
```

Comment:
Q: would it be better to give this new prop a name that is more about what the prop does? Something like `forceShow` ?